build_library/generate_grub_hashes.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python
```

Comment:
I'm guessing some day we will want to combine this with our grub install script or have a generalized implementation in grub itself so we can generate this directly from grub.cfg, avoiding accidental mismatches when things change. Short of that we are going to need some build time check and/or kola test to make sure we don't accidentally make a release with this script and grub.cfg conflicting.